12 TestAction/sample.test.ts

```diff
@@ -0,0 +1,20 @@
+describe('Sample tests', () => {
+    it('should pass spec', () => {
+        // Arrange
+
+        // Act
+
+        // Assert
+        expect(true).toBeTruthy();
+    });
+
+
+    it('should fail spec', () => {
+           // Arrange
+
+               // Act
+
+               // Assert
+               expect(false).toBeTruthy();
+         });
```

Comment:
Mmm this test should be commented out or something like that (we don't want a test to fail)

Reply:
removed file. Was a basically a testing case scaffold